dts/bindings/tach/realtek,rts5912-tach.yaml

```diff
@@ -0,0 +1,28 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) 2025 Realtek Semiconductor Corporation, SIBG-SD7
+#
+
+description: Realtek rts5912 tachometer controller
+
+compatible: "realtek,rts5912-tach"
```

Comment:
I think I'm seeing what my confusion here was, this is a tachometer that's specific to the rts5912? Is that part not available separately? I can't seem to find any information past https://www.realtek.com/Product/ProductHitsDetail?id=4422&menu_id=419&lang=en-GB.

Is this a proprietary driver?